app/check_data_type.py

```diff
@@ -0,0 +1,19 @@
+"""
+filename: check_data_type.py
+description: Checks if incoming data is a dictionary.
+created by: Omar De La Hoz (oed7416@rit.edu)
+created on: 09/13/18
+"""
+import functools
+from flask_socketio import emit
+from flask import request
+
+def checkDict(types):
```

Comment:
I think you should rename this to be more descriptive of what its doing, maybe something like "ensure_dict" or "is_dict", also generally python function and variable names are not camelcased, they are underscored separated.

Reply:
Renamed to ensure_dict